.spelling

```diff
@@ -242,7 +244,8 @@ iLOM
 IMS
 IMS-hosted
 inband
-Infiniband
+InfiniBand
+Init
```

Comment:
Since this is only for the phrase Site Init, and since in general Init would be invalid, I suggest instead adding this as a per-file override. Delete this line and at the end of the .spelling file add the following lines:
```text
- glossary.md
Init

 - install/prepare_configuration_payload.md
Init

 - introduction/site_survey.md
Init
```

```suggestion
```

Reply:
Adding.